exercises/practice/sum-of-multiples/.meta/tests.toml

```diff
@@ -0,0 +1 @@
+
```

Comment:
This shouldn't be blank so that we can keep track of new/changed/removed test cases in the future. You can generate it interactively with `configlet sync --update`, or, if you've copied all cases from problem-specs, `configlet sync --tests include --exercise sum-of-multiples --update`.

https://github.com/exercism/problem-specifications/#track-test-data has more info.

Reply:
Thanks for the guidance. It should be fixed now.